Recipes/definitions.recipe.json

```diff
@@ -2513,53 +2527,13 @@
                 "ContentIndexSettings": {}
               }
             },
-            {
-              "FieldName": "TextField",
-              "Name": "ContentWidth",
-              "Settings": {

```

Comment:
Why is this field being removed? I think it was recently added and probably something we want to keep?

Reply:
Okay - approving with suggestion because as long as this gets resolved I'm happy for it to be merged. :)